LDAP/disable_deactivated_accounts.py

```diff
@@ -107,15 +116,33 @@ def disable_user(user_id):
             print "%s %s" % (resp.status_code, resp.text)
     return True
 
+# this function does the unblocking of the user in auth0
+def unblock_user(user_id):
+    deactive_url = "users/%s" % user_id
+    url = "%s/%s" % (auth0_config['auth0_api_url'], deactive_url)
+    body = '{"blocked":false}'
+    if proxy_config['use_proxy'] is True:
+        resp = requests.patch(url, headers=build_headers(), data=body, proxies=proxy_config['proxies'])
+        if resp.status_code != 200:
```

Comment:
shouldnt this also fail loudly in some way? or maybe thats the way we have atm
generally i'd except something like
`raise Exception unblockFailure`
or some alert somewhere

Reply:
@jdow ^^

Reply:
I think the intention here is that it doesn't cause the script to blow up and stop running if there's a failure here. Since this function gets called in a loop with potentially lots of users that are queued to get blocked or unblocked, a failure in one of them shouldn't stop the rest from being processed. Since it runs from cron, the cronmail would show the same message each time it runs and would nag. If I raise an exception here, wouldn't that stop the script from finishing the rest of the blocks/unblocks until it is dealt with manually?

Reply:
yes it would stop the script
i dont know if thats the best way to handle it but ill just unblock this pr for now...